Hero/Hero.swift

```diff
@@ -166,18 +175,19 @@ internal extension Hero {
     // wait for a frame if using navigation controller.
     // a bug with navigation controller. the snapshot is not captured if animating immediately
     delay(inContainerController && presenting ? 0.02 : 0) {
-      for (currentFromViews, currentToViews) in self.animatorViews {
+      self.animatorViews.forEach {
         // auto hide all animated views
-        for v in currentFromViews {
-          self.context.hide(view: v)
+        $0.0.forEach {
+          self.context.hide(view: $0)
         }
-        for v in currentToViews {
-          self.context.hide(view: v)
+        
+        $0.1.forEach {
+          self.context.hide(view: $0)
```

Comment:
Don't think we should replace these for loops with forEach. Makes some of the code harder to read. And less clear to people who are not familiar with functional programming. Might be slower too.